data/persons.txt

```diff
@@ -807,3 +807,82 @@ person "Subsidurial"
 		"never disabled"
 	system
 		government "Uninhabited"
+
+person "MCOfficer"
+	frequency 200
+	government "Author"
+	personality
+		disables heroic plunders opportunistic
+	phrase
```

Comment:
@MCOfficer hasn't provided any special dialog as of yet.

Reply:
I can come up with some if you give me some time.

Reply:
I don't know of unique dialog is a requirement for a person, nor if this is a blocking issue. Either way, it'll be more interesting with custom dialog; but it could be added later pretty easily.